src/datatypes/interval.coffee

```diff
@@ -197,6 +197,22 @@ module.exports.Interval = class Interval
     catch
       false
 
+  starts: (other, precision) ->
+    if precision? and @low instanceof DateTime
+      startEqual = @low.sameAs(other.low, precision)
+    else
+      startEqual = cmp.equals(@low, other.low)
+    lessThanOrEqual = cmp.lessThanOrEquals(@high, other.high, precision)
```

Comment:
I suggest changing this variable name to follow same convention as in ends, e.g. endLessThanOrEqual. Obviously this is just for symmetry and ease of maintenance.